temporalcli/commands.taskqueue_get_build_id.go

```diff
@@ -0,0 +1,126 @@
+package temporalcli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalTaskQueueGetBuildIdsCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	sets, err := cl.GetWorkerBuildIdCompatibility(cctx, &client.GetWorkerBuildIdCompatibilityOptions{
+		TaskQueue: c.TaskQueue,
+		MaxSets:   c.MaxSets,
+	})
+	if err != nil {
+		return fmt.Errorf("unable to get task queue build ids: %w", err)
+	}
+
+	type rowtype struct {
+		BuildIds      []string
+		DefaultForSet string
+		IsDefaultSet  bool
+	}
+	var items []rowtype
+	for ix, s := range sets.Sets {
+		row := rowtype{
+			BuildIds:      s.BuildIDs,
+			IsDefaultSet:  ix == len(sets.Sets)-1,
+			DefaultForSet: s.BuildIDs[len(s.BuildIDs)-1],
+		}
+		items = append(items, row)
+	}
+
+	if cctx.JSONOutput {
+		return cctx.Printer.PrintStructured(items, printer.StructuredOptions{})
+	}
```

Comment:
Can just get rid of this. `Println` doesn't print when JSON set, and `Table` is ignored when JSON set, so the code below will do what you want. So basically `PrintStructured` is abstracted for reuse for just this kind of use case.

Reply:
Great, much cleaner thanks!